src/web/handlers/GestureHandler.ts

```diff
@@ -736,6 +737,14 @@ export default abstract class GestureHandler {
     return false;
   }
 
+  protected wasPressedButtonCorrect(mouseButton: MouseButton | undefined) {
```

Comment:
Not sure about the name:
1. I'd change it to the present tense to keep it consistent
2. `correct` sounds strange to me in this context, maybe something along `matches config`

Idk, I can be convinced I'm wrong here.

Reply:
What about `isButtonInConfig`?

Reply:
Sounds good

Reply:
Done in [09e11fd](https://github.com/software-mansion/react-native-gesture-handler/pull/2676/commits/09e11fdf9a0e9ea06446a026ed4a5bf68058b745)